portal/models/message.py

```diff
@@ -25,7 +25,31 @@ def log_message(message, app):
     "<html><head><title>TrueNTH email</title><style>"
     "body {"
     " font-size: 16px;"
-    "}"
+    "} "
```

Comment:
Getting a bit long there, think it makes sense to move into a separate jinja template yet?

Reply:
Yea, I'll move this out to a jinja template as well.

Reply:
Actually, not only does this cause the job to run slower (as we have to pass a whole bunch of generated email html through a `render_template` call), but I'm having issues with the styling getting dropped when it's moved out to a template.

I'm moving the reporting table out to a template, but I think we can just leave this emailmessage styling here for now (it's all in one chunk, like the intervention_strategies html, so it's still easily readable/understandable). If it gets more bloated, we can look at ways to abstract it out to a template.